pt-br/logistics/README.md

```diff
@@ -0,0 +1,71 @@
+# Logística
+
+Há um par de coisas que são importantes durante a organização de um evento Django Girls. Não vamos tentar explicar tudo nos mínimos detalhes, mas nós fazemos a elaboração sobre um par das coisas mais importantes:
```

Comment:
Acho que esse termo "um par" (que era "a couple of" certo?) fica meio estranho. Eu colocaria na segunda frase algo como ...elaboração de algumas coisas mais importantes